stock_picking_auto_create_lot_serial_only/tests/test_constraint.py

```diff
@@ -1,21 +1,15 @@
 # Copyright 2020 ACSONE SA/NV
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 from odoo.exceptions import ValidationError
-from odoo.tests import SavepointCase
+from odoo.tests import TransactionCase
 
 from odoo.addons.stock_picking_auto_create_lot.tests.common import (
     CommonStockPickingAutoCreateLot,
 )
 
 
-class TestProductAutoLotConstraint(CommonStockPickingAutoCreateLot, SavepointCase):
+class TestProductAutoLotConstraint(CommonStockPickingAutoCreateLot, TransactionCase):
     def test_constraint_product(self):
         # Product constraint
         with self.assertRaises(ValidationError):
             self._create_product()
-
-    def test_constraint_picking_type(self):
```

Comment:
@ThomasBinsfeld Why removing this tests ??

Reply:
Because it seems irrelevant. Changing this field on the picking type is not supposed to raise a ValidationError.